dagshub_annotation_converter/formats/cvat/__init__.py

```diff
@@ -0,0 +1,18 @@
+from typing import Callable
+
+from lxml.etree import ElementBase
+
+from .box import parse_box
+from .polygon import parse_polygon
+from .points import parse_points
+from .skeleton import parse_skeleton
+from dagshub_annotation_converter.ir.image import IRAnnotationBase
+
+CVATParserFunction = Callable[[ElementBase, ElementBase], IRAnnotationBase]
```

Comment:
You probably want to rename this module to `cvat/xml2` to disambiguate

Reply:
I'm out of the loop here, can you let me know what's the thing it's supposed to disambiguate with? They have multiple formats?

Reply:
Yes they can output to multiple formats, and the one we support AFAIK is "CVAT for images" xml, I guess 1.1 not 2.0 actually
https://docs.cvat.ai/docs/manual/advanced/formats/